### DISCUSSION TITLE
Tag location in the input string

### DISCUSSION THREAD

**Discussion by jazz-soft** [NaturalIntelligence/fast-xml-parser#686] [Ideas]
Is it possible to attach tag location (from, to) in the input string to the output tree nodes?
That would be very useful in GUI apps with syntax highlighting.

**Comment by amitguptagwl** [comment]
do you mean the line and column number? You can use attributes for that purpose if possible.

**Reply by jazz-soft** [reply]
Which option should I pass to XMLParser to add line/column information to the tags?
Sorry, could not find in the docs...

**Reply by amitguptagwl** [reply]
Currently, there is no option of getting line or column number. But it can be a good idea to add meta detail about a tag like location information. However, adding this functionality may break many project currently using this library. So I may plan it for v5.

**Comment by amitguptagwl** [comment]
This feature is implemented by Steven in v5.2.0
